src/components/flipper.ts

```diff
@@ -206,23 +206,19 @@ export default class Flipper {
    * @returns {boolean}
    */
   private isEventReadyForHandling(event: KeyboardEvent): boolean {
-    const handlingKeyCodeList = [
-      _.keyCodes.TAB,
-      _.keyCodes.ENTER,
-    ];
-
     const isCurrentItemIsFocusedInput = this.iterator.currentItem == document.activeElement;
```

Comment:
do we need this statement in the new logic? Looks like it was used to handle similar problem with arrows in inputs, but it does not cover the case when you allows up/down but disallow left/right.